homework/01LessonOne.js

```diff
@@ -144,7 +145,9 @@ var notNotUndefined;
 // Explain truthiness and falsiness in your own words. Provide an example for each.
 //
 // Your Answer Goes Here:
-//
+//truthiness and falsiness are when things may not be ABSOLUTELY true or false, but fall more toward one of those absolutes than the other.
+//truthiness example: !null
+//falsiness example: !!undefined
 // ****
```

Comment:
Close - 'truthy' and 'falsey' has to do with how Booleans are derived from a value when it's **evaluated**. `null` and `undefined` **are themselves falsey** - when you evaluate them in an expression such as `!!undefined`, they produce `false`.